yarn-endpoints-behavior.html

```diff
@@ -102,6 +102,15 @@
       },
 
       /**
+       * The encoded request path built from the computed URL
+       */
+      path: {
+        type: 'String',
+        readOnly: true,
```

Comment:
You have to use `this._setPath()` to change the prop.  See https://www.polymer-project.org/1.0/docs/devguide/properties.html#read-only.